docs/src/appendix/cheatcodes/cheat_target.md

```diff
@@ -16,28 +16,3 @@ enum CheatTarget {
 > 📝 **Note**
 > 
 > `CheatTarget::Multiple` acts as a helper for targeting every specified address separately with `CheatTarget::One`.
-> It does not treat the list of specified addresses as a group unit.
```

Comment:
Why remove this part?

Reply:
I think the sentence above explains it pretty well, the one below only introduces some "group unit" term which might be confusing.

Reply:
Sometimes, the simpler - the better